esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2214,8 +2229,37 @@ bool CFileSprayEx::onReplicate(IEspContext &context, IEspReplicate &req, IEspRep
     return true;
 }
 
+void CFileSprayEx::getDropZoneInfoByDestPlane(double clientVersion, const char* destGroup, const char* destFileIn, StringBuffer& destFileOut, StringBuffer& umask)
+{
+#ifdef _CONTAINERIZED
+    const bool isDestAbsolutePath = isAbsolutePath(destFileIn);
+    Owned<IPropertyTreeIterator> planes = queryDropZonePlanesIterator(destGroup);
+    ForEach(*planes)
+    {
+        IPropertyTree & plane = planes->query();
+        StringBuffer fullDropZoneDir(plane.queryProp("@prefix"));
+        addPathSepChar(fullDropZoneDir);
+        if (isDestAbsolutePath)
+        {
+            if (strncmp(fullDropZoneDir, destFileIn, fullDropZoneDir.length())==0)
+                destFileOut.set(destFileIn);
+            else
+                continue;
+        }
+        else
+        {
+            destFileOut.set(fullDropZoneDir).append(destFileIn);
+        }
+        plane.getProp("@umask", umask);
+        return;
+    }
+    throw MakeStringException(ECLWATCH_DROP_ZONE_NOT_FOUND, "No drop zone configured for %s:%s", destGroup, destFileIn);
```

Comment:
trivial: makeStringException/makeStingExceptionV preferred, should consider MakeStringException deprecated